src/disabled_disk_io.cpp

```diff
@@ -119,7 +119,7 @@ struct TORRENT_EXTRA_EXPORT disabled_disk_io final
 		, std::function<void(status_t, std::string const&, storage_error const&)> handler) override
 	{
 		post(m_ios, [h = std::move(handler), path = std::move(p)] () mutable
-			{ h(status_t::no_error, std::move(path), storage_error{}); });
```

Comment:
I don't see how this is useless. the lambda is `mutable`, `path` should be too.

Reply:
This was all automated. No idea what clang-tidy was thinking.